tests/initcoder_test.go

```diff
@@ -0,0 +1,78 @@
+package tests
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/Fantom-foundation/go-opera/tests/contracts/initcode"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInitCodeSizeLimitAndMetered(t *testing.T) {
+	r := require.New(t)
+
+	net, err := StartIntegrationTestNet(t.TempDir())
+	r.NoError(err)
+	defer net.Stop()
+
+	// Deploy the invalid start contract.
+	contract, receipt, err := DeployContract(net, initcode.DeployInitcode)
+	r.NoError(err)
+	r.Equal(types.ReceiptStatusSuccessful, receipt.Status, "failed to deploy contract")
+
+	const gasForOneWord uint64 = 54977
```

Comment:
where does this come from?

Reply:
I'm changing the test to better show where this value comes from.